### DISCUSSION TITLE
problem with "include_all_commits"

### DISCUSSION THREAD

**Discussion by Victor-Palha** [anuraghazra/github-readme-stats#1944] [Q&A] [answered]
Using `include_all_commits=true` on url make no one commits appears, i have to change to `false`. Until yesterday was working normal

**Comment by rickstaa** [comment, marked as answer]
@Victor-Palha Thanks for your question. I tested our stats card with your profile and I can not reproduce the bug. Maybe you made a typo in the syntax.

**With `include_all_commits=true`**

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Victor-Palha&include_all_commits=true)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Victor-Palha&include_all_commits=true)](https://github.com/anuraghazra/github-readme-stats)
```
**With `include_all_commits=false`**

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Victor-Palha&include_all_commits=false)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Victor-Palha&include_all_commits=false)](https://github.com/anuraghazra/github-readme-stats)
```

**Without `include_all_commits`**

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Victor-Palha)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Victor-Palha)](https://github.com/anuraghazra/github-readme-stats)
```

**Reply by Victor-Palha** [reply]
I tested and work normally in this morning, but yesterday was not working... Weird kk

**Reply by rickstaa** [reply]
No problem, great to hear your problem is fixed. Feel free to let me know if it happens again.
